homeassistant/components/kegtron/sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Support for Kegtron sensors."""
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from kegtron_ble import (
+    SensorDeviceClass as KegtronSensorDeviceClass,
+    SensorUpdate,
+    Units,
+)
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import SIGNAL_STRENGTH_DECIBELS_MILLIWATT, VOLUME_LITERS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .device import device_key_to_bluetooth_entity_key, sensor_device_info_to_hass
+
+SENSOR_DESCRIPTIONS = {
+    KegtronSensorDeviceClass.PORT_COUNT: SensorEntityDescription(
+        key=f"{KegtronSensorDeviceClass.PORT_COUNT}",
+        icon="mdi:water-pump",
+    ),
+    KegtronSensorDeviceClass.KEG_SIZE: SensorEntityDescription(
+        key=f"{KegtronSensorDeviceClass.KEG_SIZE}",
```

Comment:
I _think_ it will work if you drop the format string wrapping here since it's a SteEnum but if mypy complains it might need to be wrapped in str() and then I'm not sure it's worth changing

Reply:
mypy isn't complaining on my end, so changed it anyway.